### PR TITLE
feat(LoginPage): enable show or hide password

### PR DESCRIPTION
**Pull request by nicolethoen**
Closes: #5422

### CODE REVIEW

**Review by mcarrano — COMMENTED** [review]
Looks great @nicolethoen . Just missing the screenshot to link to the full page example.

**Review by mcoker — APPROVED** [review]
LGTM! 🥳

**Comment by nicolethoen** [comment]
@redallen is there a command to run to generate and add full page screen shots?

**Comment by redallen** [comment]
`yarn build:docs` -> `yarn serve:docs` -> (another terminal) `yarn workspace @patternfly/react-docs screenshots`.

This won't transfer to patternfly.org, though. It's most important to update the screenshots there every release.

**Comment by nicolethoen** [comment]
@mcarrano I added updating example screenshots to my release to-do list. So that'll happen at release.

**Review by jessiehuff — DISMISSED** [review]
LGTM! 🙂

**Review by mcarrano — DISMISSED** [review]
> @mcarrano I added updating example screenshots to my release to-do list. So that'll happen at release.

I will approve then.

File: packages/react-core/src/components/LoginPage/LoginForm.tsx [hunk 1]
```diff
@@ -32,6 +35,12 @@ export interface LoginFormProps extends React.HTMLProps<HTMLFormElement> {
   onChangePassword?: (value: string, event: React.FormEvent<HTMLInputElement>) => void;
   /** Flag indicating if the Password is valid */
   isValidPassword?: boolean;
+  /** Flag indicating if the user can toggle hiding the password */
+  isShowPasswordEnabled?: boolean;
+  /** Label for the show password button */
+  showPasswordLabel?: string;
```
Comment (tlabaj) [inline packages/react-core/src/components/LoginPage/LoginForm.tsx:41]: It looks like this prop and the one below are used for aria-label.  If that is the case I suggest `showPasswordAriaLabel` and `hidePasswordAriaLabel

File: packages/react-core/src/components/LoginPage/LoginForm.tsx [hunk 2]
```diff
@@ -67,59 +79,84 @@ export const LoginForm: React.FunctionComponent<LoginFormProps> = ({
...
+              variant="control"
+              onClick={() => setPasswordHidden(!passwordHidden)}
+              aria-label={passwordHidden ? showPasswordLabel : hidePasswordLabel}
+            >
+              {passwordHidden ? <EyeIcon /> : <EyeSlashIcon />}
+            </Button>
+          </InputGroup>
+        )}
+        {!isShowPasswordEnabled && (
+          <TextInput
```
Comment (tlabaj) [inline packages/react-core/src/components/LoginPage/LoginForm.tsx:133]: this is pretty much duplicate of lines 114-122.  can you make cont of that code and use that instead?

**Review by tlabaj — CHANGES_REQUESTED** [review]
just  a couple of comments

**Review by tlabaj — APPROVED** [review]

**Review by jessiehuff — APPROVED** [review]

**Review by mcoker — APPROVED** [review]

**Pull request merged**